pages/[...slug].tsx

```diff
@@ -116,20 +117,30 @@ export async function getStaticProps(context: GetStaticPropsContext): Promise<Ge
 export default function Page({ node, nav, footer }: PageProps) {
   const router = useRouter()
   const [cookieConsent] = useState<string>(getCookieConsentValue('tyollisyyspalvelut_cookie_consent'))
-  useReactAndShare(cookieConsent, router.locale, node && node.title)
 
   if (!router.isFallback && !node?.id) {
     return <ErrorPage statusCode={404} />
   }
 
   if (!node) return null
 
+  const { t } = useTranslation('common')
+  const metaTitle = node.field_metatags?.title ? node.field_metatags.title : node.title
+  const metaDescription = node.field_metatags?.description ? node.field_metatags.description : node.field_lead_in
+  const metaUrl = process.env.NEXT_PUBLIC_SITE_URL + router.asPath
+  const metaImage = node?.field_image_url ? node.field_image_url : process.env.NEXT_PUBLIC_SITE_URL + "/tyollisyyspalvelut.png"
```

Comment:
Seems that only Event content type is using `field_image_url`. You could also add metaimage support for Landing page image that means getting the hero image if it's set on Landing page (usually landing page should use hero with image).

Add also metatags for /pages/index.js so the frontpage will have them also.

Reply:
Added Meta info for index page. Updated image information per content type.